N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
I do not understand the comment.

The project is called is called N3.js, hence `N3` being used here. `N3.js` can actually parse N3. The current implementation focuses on Turtle and the types are not intended to lie here.

The fact that this PR doesn't expose **all** the features from N3.js and jsonld.js is a different matter. The quad things is a wanted feature but please stay focused on this PR for now. If that PR doesn't prevent one from doing what you are talking about in the future, then this is just good to go.